### DISCUSSION TITLE
LGTMボタンだけ緑色なのでクリック済みと勘違いしてしまう

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#168] [要望・アイデア・不具合]
記事の左サイドに表示されているボタン類の中で、LGTMボタンだけ最初から緑色なのが気になっています。
「1つだけ緑色＝このボタンだけクリック済み」と認識してしまうので、記事を見たときに一瞬「あれ、この記事LGTMしたっけ？」と勘違いしてしまいます（ストックボタンもクリックしたら緑色になるのかな、と思ってしまう）。

<img width="89" alt="Screen Shot 2022-03-08 at 11 47 02" src="https://user-images.githubusercontent.com/1148320/157156766-a5235c76-bff1-449b-a073-42d44ffc8d51.png">

いちおう参考までにクリック済みの状態も載せておきます。

<img width="99" alt="Screen Shot 2022-03-08 at 11 46 52" src="https://user-images.githubusercontent.com/1148320/157156716-b6935d02-7eed-4d45-b2fb-6541c885e49a.png">

どういうふうに改善したら良いのかはちょっとアイデアが思いつきませんが、何かしらデザイン（色遣い？）の改善をしてもらえたら嬉しいです。

**Comment by kskwtnk** [comment]
@JunichiIto 
ご意見ありがとうございます！

具体的にどう変えるかは未知数ですが、1度検討してみます！

余談となってしまいますが、言われてみれば多くのメディア系のサイトは黒やグレーなどニュートラルな色ですね

このUIになった当時はまだ私がQiita社に在籍していなかったので予想ですが……
プライマリーなボタンなので緑を採用したのかと思い、それはそれで理解できるものの、改めて他サービスも見るとレアケースでした
中の人だと余計に気付きづらい観点でして、ありがとうございます 🙏

**Comment by tanaka-asami415** [comment]
@JunichiIto 
大変お待たせしました🙏

クリック済みとそうでない時の違いが分かりやすいよう、
「LGTM」から「いいね」に戻すことと合わせて、デザインを変更いたしました！✨
[Qiitaの「LGTM」を「いいね」に戻しました #283](https://github.com/increments/qiita-discussions/discussions/283)
![like-icon](https://user-images.githubusercontent.com/103916185/186571121-7c7d6b88-9227-4e10-a4bf-4b2236068a17.png)
